packages/quantic/force-app/main/default/lwc/quanticTabBar/e2e/pageObject.ts

```diff
@@ -0,0 +1,76 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaSearchEvent} from '../../../../../../playwright/utils/requests';
+
+export class TabBarObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get tabBar(): Locator {
+    return this.page.locator('c-quantic-tab-bar');
+  }
+
+  get allVisibleTabs(): Locator {
+    return this.tabBar.locator('c-quantic-tab').filter({
+      has: this.page.locator(':visible'),
+    });
+  }
+
+  get activeTab(): Locator {
+    return this.tabBar.locator('c-quantic-tab').filter({
+      has: this.page.locator('.slds-is-active'),
+    });
+  }
+
+  get moreButton(): Locator {
+    return this.tabBar.getByTestId('tab-bar_more-button');
```

Comment:
That didn't work with the `getByRole` right? I think I remember trying to make it work...

Reply:
yeah I tried it but it worked with `getByTestId`